service/internal/cloudconfig/tccpn.go

```diff
@@ -21,6 +24,8 @@ import (
 	"github.com/giantswarm/aws-operator/service/internal/hamaster"
 )
 
+const keyCloudPrefix = "tag.provider.giantswarm.io/"
```

Comment:
Shall we move this to https://github.com/giantswarm/aws-operator/blob/master/service/controller/key/common.go#L18? WDYT?